include/savedata_misc.h

```diff
@@ -0,0 +1,58 @@
+#ifndef POKEPLATINUM_SAVEDATA_MISC_H
+#define POKEPLATINUM_SAVEDATA_MISC_H
+
+#include "struct_defs/sentence.h"
+#include "strbuf.h"
+#include "struct_decls/struct_02027854_decl.h"
+#include "struct_decls/struct_02027860_decl.h"
+#include "struct_defs/struct_02027854.h"
+#include "struct_defs/struct_02027860.h"
+#include "savedata.h"
+#include "constants/savedata/save_table.h"
+#include "constants/savedata/savedata_misc.h"
+
+typedef struct ExtraSaveKey {
+    u32 keys[EXTRA_SAVE_TABLE_ENTRY_MAX - 1];
+    u32 oldKeys[EXTRA_SAVE_TABLE_ENTRY_MAX - 1];
+    u8 keyFlags[EXTRA_SAVE_TABLE_ENTRY_MAX - 1];
+    u8 filler[3];
```

Comment:
Convention in the project is to name member-vars like these as `padding`, with a suffix if there are multiple instances of padding in the struct.